msbuild/Xamarin.MacDev.Tasks.Core/Tasks/ResolveNativeReferencesBase.cs

```diff
@@ -64,24 +64,26 @@ public override bool Execute ()
 				foreach (var nr in NativeReferences) {
 					var name = nr.ItemSpec;
 					switch (Path.GetExtension (name)) {
-					// '.' is used to represent a file (instead of the directory)
+					// '.' can be used to represent a file (instead of the directory)
 					case "":
 						name = Path.GetDirectoryName (name);
-						if (Path.GetExtension (name) == ".xcframework") {
-							var resolved = ResolveXCFramework (name);
-							if (resolved == null)
-								return false;
-							var t = new TaskItem (resolved);
-							t.SetMetadata ("Kind", "Framework");
-							t.SetMetadata ("Name", resolved);
-
-							// add some metadata from the original item
-							var addToAppBundle = nr.GetMetadata ("AddToAppBundle");
-							if (!string.IsNullOrEmpty (addToAppBundle))
-								t.SetMetadata ("AddToAppBundle", addToAppBundle);
+						if (Path.GetExtension (name) == ".xcframework")
+							goto case ".xcframework";
```

Comment:
First time seeing a `goto` always seeing something new in these PRs haha

Reply:
Could be written without a goto and pattern matching ;)